src/server/lib/audit-report.js

```diff
@@ -1,876 +1,118 @@
-/* eslint camelcase: 0 */
-
-/*
---------------------------------------------------------------------------------
--                           lib/audit-report.js
---------------------------------------------------------------------------------
-
-*/
-const _ = require('lodash')
+const { format } = require('date-fns')
 const XLSX = require('xlsx')
+
+const { get, getAll } = require('../db/reporting-periods')
```

Comment:
maybe alias them? `get` is kinda hard to grep for, compared to, like, `getReportingPeriod`

Reply:
I'd honestly be happier with just exporting the verbose name.  How does that sound to you?